src/open_inwoner/accounts/migrations/0048_auto_20221124_1515.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.15 on 2022-11-24 14:15
+
+import uuid
+
+from django.db import migrations
+
+
+def gen_user_uuid(apps, schema_editor):
+    MyModel = apps.get_model("accounts", "User")
+    for row in MyModel.objects.all():
+        row.uuid = uuid.uuid4()
+        row.save(update_fields=["uuid"])
+
+
+def populate_user_contacts(apps, schema_editor):
+    MyModel = apps.get_model("accounts", "Contact")
+    for contact in MyModel.objects.exclude(contact_user__isnull=True):
+        creator = contact.created_by
+        contact_user = contact.contact_user
+        creator.user_contacts.add(contact_user)
+
+        contact_user.first_name = contact.first_name
```

Comment:
populating user_contacts is appreciated, however I would avoid setting the first_name/last_name/phonenumber/function fields on the User in the migration.